### PR TITLE
[CI] [Chore] Add label 4.0.0 to mergeable check

### PR DESCRIPTION
**Pull request by SbloodyS**
## Brief change log

Add label 4.0.0 to mergeable check

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: .github/mergeable.yml [hunk 1]
```diff
@@ -58,5 +58,5 @@ mergeable:
               regex: 'feature|bug|improvement|document|chore'
               message: 'Label must include one of the following: `feature`, `bug`, `improvement`, `document`, `chore`'
           - must_include:
-              regex: '3\.0\.x|3\.1\.x|3\.2\.0'
-              message: 'Label must include one or more of the following: `3.0.x`, `3.1.x`, `3.2.0`'
+              regex: '3\.0\.x|3\.1\.x|3\.2\.0|4\.0\.0'
```
Comment (kezhenxu94) [inline .github/mergeable.yml]: Let's change these hardcoded versions to something like `\d+.\d+.\d+`

Comment (SbloodyS) [reply .github/mergeable.yml]: I'll fix it.

File: .github/mergeable.yml [hunk 2]
```diff
@@ -58,5 +58,5 @@ mergeable:
...
+              message: 'Label must include one or more of the following: `3.0.x`, `3.1.x`, `3.2.0`, `4.0.0`'
```
Comment (kezhenxu94) [inline .github/mergeable.yml]: And change this to 
```suggestion
              message: 'Label must include one or more version numbers. 
```

File: .github/mergeable.yml (hunk 1 above)
Comment (zhongjiajie) [reply .github/mergeable.yml]: reg exp can be use according to https://mergeable.readthedocs.io/en/latest/validators/label.html#label

**Review by zhongjiajie — DISMISSED** [review]

File: .github/mergeable.yml [hunk 3]
```diff
@@ -58,5 +58,5 @@ mergeable:
               regex: 'feature|bug|improvement|document|chore'
               message: 'Label must include one of the following: `feature`, `bug`, `improvement`, `document`, `chore`'
           - must_include:
-              regex: '3\.0\.x|3\.1\.x|3\.2\.0'
-              message: 'Label must include one or more of the following: `3.0.x`, `3.1.x`, `3.2.0`'
+              regex: '\d+\.\d+\.x|\d+\.\d+\.\d+'
```
Comment (zhongjiajie) [inline .github/mergeable.yml]: And how about \w+ directly?
```suggestion
              regex: '\d+\.\d+\.\w+'
```

Comment (SbloodyS) [reply .github/mergeable.yml]: Sure.

**Review by zhongjiajie — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]

**Comment by zhongjiajie** [comment]
It seems mergeable config will flush only if we merge into dev branch

**Comment by zhongjiajie** [comment]
I think is ok to merge it and check whether the config work or not

**Comment by SbloodyS** [comment]
> It seems mergeable config will flush only if we merge into dev branch

It seems so....

**Pull request merged**